Airports/EGPH/Stars.txt

```diff
@@ -1,16 +1,15 @@
-STAR:EGPH:24:STIRA1A:PTH GRICE STIRA CF24
-STAR:EGPH:06:STIRA1A:PTH GRICE STIRA CF06
-STAR:EGPH:24:TWEED2B:BLACA TUNSO TLA TARTN TWEED CF24
-STAR:EGPH:06:TWEED2B:BLACA TUNSO TLA TARTN TWEED CF06
-STAR:EGPH:24:TWEED2C:BLACA GIRVA TLA TARTN TWEED CF24
-STAR:EGPH:06:TWEED2C:BLACA GIRVA TLA TARTN TWEED CF06
-STAR:EGPH:24:TWEED2D:HAVEN TARTN TWEED CF24
-STAR:EGPH:06:TWEED2D:HAVEN TARTN TWEED CF06
-STAR:EGPH:24:TWEED3A:ESKDO PHS48 TARTN TWEED CF24
-STAR:EGPH:06:TWEED3A:ESKDO PHS48 TARTN TWEED CF06
-STAR:EGPH:24:EDN3A:ESKDO PHS48 TARTN EDN CF24
-STAR:EGPH:06:EDN3A:ESKDO PHS48 TARTN EDN CF06
-STAR:EGPH:24:EDN1D:PTH GRICE UW EDN CF24
-STAR:EGPH:06:EDN1D:PTH GRICE UW EDN CF06
-STAR:EGPH:24:EDN2E:HAVEN TARTN EDN CF24
-STAR:EGPH:06:EDN2E:HAVEN TARTN EDN CF06
+STAR:EGPH:06:PTH1G:PTH GRICE STIRA
+STAR:EGPH:24:INPIP1E:INPIP INREV ESKDO TARTN
+STAR:EGPH:24:AGPED1E:AGPED HAVEN TARTN
+STAR:EGPH:24:GIRVA1E:GIRVA TLA TARTN
+STAR:EGPH:24:TUNSO1E:TUNSO TLA TARTN
```

Comment:
Add the relevant CF (center fix) points - just helps with the route draw for APP controllers